packages/react-native/src/primitives/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Children, cloneElement, useMemo, useState } from 'react';
+import { View } from 'react-native';
+
+import { childIsValidComponent } from '../../utils';
+import { Button } from '../Button';
+
+import { styles } from './styles';
+import { TabProps, TabsProps } from './types';
+
+export function Tab({ title, ...rest }: TabProps): JSX.Element {
```

Comment:
It's better practice to have one file per component for improved readability and separation of concers.